.github/workflows/codeql-analysis.yml

```diff
@@ -32,7 +32,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  Z3_VERSION: 4.10.0
+  Z3_VERSION: 4.13.0 # latest version of Z3
```

Comment:
These comments will go out of sync quite quickly. Namely, as soon as 4.14 is released. I would rather remove them